packages/good-design/src/apps/goodid/wizards/SegmentationWizard.tsx

```diff
@@ -104,6 +105,7 @@ export const SegmentationWizard = (props: SegmentationProps) => {
     try {
       await props.onLocationRequest(...args);
     } catch (e: any) {
+      track("goodid_error", { error: "SEGMENTATION_FAILED", message: e?.message, e });
```

Comment:
give better error names instead of using the same one. for example this one here is onLocationRequest failed not SEGMENTATION_FAILED